scripts/run_tasks.py

```
@@ -0,0 +1,196 @@
+    """Run the external process for a task.
+
+    The command_template can include placeholders:
+      - {id}: task id (e.g., T013)
+      - {attempt}: current attempt number (1-based)
+      - {file}: absolute path to the tasks.md file
+
+    When use_shell=False, the command string is split with shlex.split.
+    """
+    try:
+        rendered = command_template.format(id=task_id, attempt=attempt, file=file_path)
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)